packages/react-native-bridge/android/src/main/java/org/wordpress/mobile/WPAndroidGlue/WPAndroidGlueCode.java

```diff
@@ -316,6 +316,11 @@ public void requestImageUploadCancel(int mediaId) {
                 mOnMediaLibraryButtonListener.onCancelUploadForMediaDueToDeletedBlock(mediaId);
             }
 
+            @Override
```

Comment:
Hi @SiobhyB we could also add this override to the [MainApplication](https://github.com/WordPress/gutenberg/blob/702c76e9e9b0de4f96b324f017716afa333f421a/packages/react-native-editor/android/app/src/main/java/com/gutenberg/MainApplication.java) class of the demo project if it's missing. Let me know if you get the project to build in Android Studio. :)

Reply:
Added the missing function to MainApplication.java. :)